custom-modifiers/waitforelementtobevisiblemodifier/src/main/java/WaitForElementToBeVisibleModifier.java

```diff
@@ -0,0 +1,77 @@
+import com.cognifide.aet.communication.api.metadata.CollectorStepResult;
+import com.cognifide.aet.job.api.collector.CollectorJob;
+import com.cognifide.aet.job.api.exceptions.ParametersException;
+import com.cognifide.aet.job.api.exceptions.ProcessingException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.concurrent.TimeUnit;
+
+public class WaitForElementToBeVisibleModifier implements CollectorJob {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(WaitForElementToBeVisibleModifier.class);
+
+    static final String NAME = "wait-for-element-to-be-visible";
+
+    private final WebDriver webDriver;
+
+    private static final String CSS_PARAMETER = "css";
```

Comment:
Please provide the documentation for new modifiers (sorry, full tutorial how to do that is in progress):
Create a proper markdown document under [`aet/documentation/src/main/wiki/`](https://github.com/Cognifide/aet/tree/master/documentation/src/main/wiki). See how other Modifier docs look like (e.g. [`ClickModifier.md`](https://github.com/Cognifide/aet/blob/master/documentation/src/main/wiki/ClickModifier.md)).
Remember to add it to wiki ToC [`_Sidebar.md`](https://github.com/Cognifide/aet/blob/master/documentation/src/main/wiki/_Sidebar.md) and to [`DocumentationTemplate.md`](https://github.com/Cognifide/aet/blob/master/documentation/src/main/DocumentationTemplate.md) in order to archive it with future releases.